packages/governance/src/types.js

```diff
@@ -20,8 +20,8 @@
  */
 
 /**
- * @typedef { 'amount' | 'brand' | 'instance' | 'installation' | 'invitation' |
- *   'nat' | 'ratio' | 'string' | 'unknown' } ParamType
+ * @typedef { 'amount' | 'brand' | 'brandedAmount'| 'brandedRatio'  | 'instance' | 'installation' | 'invitation' |
+ *   'nat' | 'ratio' | 'relativeTime' | 'string' | 'unknown' } ParamType
```

Comment:
`relativeTime` is a good idea, but it immediately makes me ask for `timeStamp`.

Reply:
I had just copied that from an earlier union. After 5c7e4d2a39cb568bda4408dc7fd8c2aa16b24294 I saw that it wouldn't work so I removed in cdb34071444888377d13123d85d642668e246af6